### ISSUE TITLE
[1.12] Auto-Workbench Crash with Crafttweaker custom blueprint

### ISSUE THREAD

**Issue by Golrith** [BluSunrize/ImmersiveEngineering#3339]
### Description of the issue:
Ive made a series of custom blueprints via CraftTweaker, but the Auto-Workbench crashes when I try to select the IE conveyor belt.

var IEBelt = <immersiveengineering:conveyor>.withTag({conveyorType: "immersiveengineering:conveyor"});
mods.immersiveengineering.Blueprint.addRecipe("Logistics", IEBelt*2,[<thermalfoundation:material:32>,<thermalfoundation:material:24>]);

### Crashlog:
https://pastebin.com/embed_iframe/fTy5h75d

### Versions & Modlist
ImmersiveEngineering-0.12-87.jar

I'm wondering if the new render system is having issues with item IDs referenced by NBT tags?

**Comment by BluSunrize** [comment]
It's a crash in the render, which is quite unusual.

I'll take a look when I have the time

**Comment by mindforger** [comment]
What is the mod vanillafix for? Other than that, can you show a general screenshot of that area, because you either have an item spillage or many conveyors close to the crash location

**Comment by Golrith** [comment]
VanillaFix is a mod that improves the performance of Minecraft, I think it makes some changes that forge team feel are a bit "dodgy", butit does provide excellent results. https://minecraft.curseforge.com/projects/vanillafix

If I comment out the offending line from the script, then the game save will load fine as the workbench no longer tries to render the object. I added a different item (also using a non-block model) from a different mod, but that also resulted in a crash, which is unusual, as I already have recipe in place for another custom model which works fine (the render shows the flat texture sheet).

That log is a report from a player of my mod pack, which I've been able to replicate in my testing world. 

Confirmed I think it's the fancy blueprint render on the workbench not liking certain blocks.
